typescript/sdk/src/ism/EvmIsmModule.ts

```diff
@@ -310,27 +263,27 @@ export class EvmIsmModule extends HyperlaneModule<
       });
 
       const domainId = this.multiProvider.getDomainId(origin);
+      const tx = await contract.populateTransaction.set(domainId, ism.address);
       updateTxs.push({
-        annotation: `Setting new ISM for origin ${origin}...`,
         chainId: this.domainId,
-        to: this.args.addresses.deployedIsm,
-        data: routingIsmInterface.encodeFunctionData('set(uint32,address)', [
-          domainId,
-          ism.address,
-        ]),
+        annotation: `Setting new ISM for origin ${origin}...`,
+        ...tx,
```

Comment:
There's a chainId inside of PopulatedTransaction so it will overwrite the one above. is this generally acceptable/desirable? 

maybe move the `chainId: this.domainId` to below it?